### PR TITLE
[KBFS-3455] Sort badged conversations first

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

File: shared/chat/inbox/container/remote.js [hunk 1]
```diff
@@ -17,17 +17,22 @@ export type RemoteConvMeta = $Diff<
...
-    .partialSort(maxShownConversations, (a, b) => b.timestamp - a.timestamp)
+    .map(v => ({
+      hasBadge: badgeMap.get(v.conversationIDKey, 0) > 0,
+      hasUnread: unreadMap.get(v.conversationIDKey, 0) > 0,
+      conversation: v,
+    }))
+    .partialSort(maxShownConversations, (a, b) => {
+      return a.hasBadge
+        ? b.hasBadge
+          ? b.conversation.timestamp - a.conversation.timestamp
```
[the remark this answers is not in the document]
Comment (jzila) [reply shared/chat/inbox/container/remote.js]: What makes you say that?

[the remark this answers is not in the document]
Comment (jzila) [reply shared/chat/inbox/container/remote.js]: We want timestamps to be sorted in descending order, not ascending.

[the remark this answers is not in the document]
Comment (songgao) [reply shared/chat/inbox/container/remote.js]: Oh that's true. My brain isn't working today!

**Comment by jzila** [comment]
Ya I pushed a fix.

**Pull request merged**